grid2op/Backend/PandaPowerBackend.py

```diff
@@ -720,17 +718,21 @@ def get_line_flow(self):
         """
         return self.a_or
 
-    def _disconnect_line(self, id):
-        if id < self._number_true_line:
-            self._grid.line["in_service"].iloc[id] = False
+    def _disconnect_line(self, id_):
+        if id_ < self._number_true_line:
+            self._grid.line["in_service"].iloc[id_] = False
         else:
-            self._grid.trafo["in_service"].iloc[id - self._number_true_line] = False
+            self._grid.trafo["in_service"].iloc[id_ - self._number_true_line] = False
+        self._topo_vect[self.line_or_pos_topo_vect[id_]] = -1
```

Comment:
Looks correct, but it will have an impact on observations I believe.
Is this due to a known bug ? Is this tested ?

Reply:
Yes a test (that i added for lightsim) did not pass without that. I don't remember which one but it was one of the one I added I should have noted it :-/
Also i think it's a convention that if an object is disconnected, then it should have "-1" at its topo_vect component. So this is consistent with this convention i belived :-)

Reply:
perfect, I was worried about some side effects.